app/services/audit_results.rb

```diff
@@ -97,12 +97,12 @@ class AuditResults
   ].freeze
 
   HONEYBADGER_REPORT_CODES = [
```

Comment:
similarly, would it be helpful to add a comment explaining why you are commenting this out (I don't have enough context to know how these codes are used)